vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -99,4 +99,28 @@ def store_ids_for_new_records(records, hashes, keys)
       h[:id] = r.id
     end
   end
+
+  # most of the refresh_multi calls follow the same pattern
+  # this pulls it out
+  def save_inventory_assoc(type, parent, hashes, target, find_key, child_keys = [], extra_keys = [])
+    reflection = parent.reflections[type]
```

Comment:
Prefer the method reflect_on_association, since it's Rails 3/4 agnostic.  Right @tenderlove ?